api/bin/pom.xml

```diff
@@ -0,0 +1,184 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
Thanks  @dexter808 ,do we nee changes here since we are only replacing .size() with isEmpty()

Reply:
No changes were made in this file.

Reply:
Hi @HerbertYiga , I did not touch these files, Maybe these changes were made by the IDE I was using. Is it a problem?

Reply:
> I did not touch these files, Maybe these changes were made by the IDE I was using. Is it a problem?

only needed changes are the ones to be merged,can you try dropping all the uneeded files for your changes

Reply:
Hi @HerbertYiga , sorry for the late reply. I will make a new PR with only the changes that are required.